app/Model/Pet.php

```diff
@@ -14,18 +15,22 @@
 class Pet extends Model
 {
     protected ?string $table = 'pets';
-
     protected array $fillable = [
         'nome',
         'data_nascimento',
+        'especie_id'
     ];
-
     protected array $casts = [
-        'id' => 'integer',
-        'nome' => 'string',
-        'data_nascimento' => 'date',
-
+        'id'=>'integer',
+        'nome'=>'string',
+        'data_nascimento'=>'date',
     ];
-
+    protected array $hidden=[
+        'created_at',
+        'updated_at'
+    ];
+    public function especie():BelongsTo{
+        return $this->belongsTo(Especie::class, 'especie_id');
```

Comment:
pro `belongsTo`, não precisa especificar a segunda parte do trecho:
```suggestion
        return $this->belongsTo(Especie::class);
```